Add argument to make cursive-style indentation (#21)
gfredericks/how-to-ns: README.md

 :import-square-brackets?          false
 :place-string-requires-at-bottom? false
 ;; if `true`, doesn't place a newline after `(:require`:
 :traditional-newline-style?       false

>  ;; if `true` make (:require\n  <add-one-more-space-symbol-here>[clojure.string :as ... 
>  ;; Only work with :traditional-newline-style? set to false

Reviewer: ```suggestion
 ;; if `true` make `(:require\n  <add-one-more-space-symbol-here>[clojure.string :as ...`
 ;; This option only has effect with `:traditional-newline-style?` set to false
```
